.github/workflows/test-doc.yml

```diff
@@ -30,8 +30,7 @@ jobs:
 
       - name: 🏗️ Install dependencies
         run: |
-          uv pip install -r pyproject.toml --group docs
-
+          uv pip install ".[docs,cpu]"
```

Comment:
@onuralpszr wdyt?

Reply:
docs not exist so answer is no it should "uv pip install -r pyproject.toml --group docs --extra cpu" 

in pypi we can only use "cpu", as I said I excluded "docs"

Reply:
@soumik12345 I'm tagging you to make sure you'll see this comment from @onuralpszr

Reply:
@onuralpszr Does this mean that we won't be able to use the cuda and rocm extras from pypi?

Reply:
very good question; @onuralpszr how will this influence regular `pip` users of the final package? will they be able to install `cpu`, `cuda`, ... ?

if not, is there any way we can unlock them?

Reply:
@SkalskiP  @soumik12345  all extras we made like "cpu", "cuda" all fine I only exclude "docs,devs,build" "--group" means for local development as "uv documentation stated. For rest we are ALL good :)